stubs/Flask-Migrate/flask_migrate/__init__.pyi

```diff
@@ -9,20 +12,33 @@ from flask_sqlalchemy import SQLAlchemy
 _T = TypeVar("_T")
 _P = ParamSpec("_P")
 _ConfigureCallback: TypeAlias = Callable[[Config], Config]
+_AlembicConfigValue: TypeAlias = Any
 
 alembic_version: tuple[int, int, int]
 log: Logger
 
 class Config:  # should inherit from alembic.config.Config which is not possible yet
     template_directory: str | None
-    def __init__(self, *args, **kwargs) -> None: ...
+    # Same as alembic.config.Config + template_directory kwarg
+    def __init__(
+        self,
+        file_: StrPath | None = None,
+        ini_section: str = "alembic",
+        output_buffer: TextIO | None = None,
+        stdout: TextIO = sys.stdout,
+        cmd_opts: Namespace | None = None,
```

Comment:
`TextIO` and `Namespace` come directly from `alembic`'s own inline types, but I have some concern about using those.
Namely if `TextIO` should be replaced by some buffer protocol.

Reply:
I haven't looked at it, but generally for I/O types in argument positions, protocols are preferred.

Reply:
I dig down a bit more.
`stdout` goes to https://github.com/sqlalchemy/alembic/blob/main/alembic/util/messaging.py#L47
`output_buffer` goes to https://github.com/sqlalchemy/alembic/blob/main/alembic/runtime/migration.py#L173